sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/SelectQuery.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Globalization;
+using Azure.DigitalTwins.Core.Queries.QueryBuilders;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.DigitalTwins.Core.QueryBuilder
+{
+    /// <summary>
+    /// Query object without any clauses.
+    /// </summary>
+    public sealed class SelectQuery : QueryBase
+    {
+        private readonly FromQuery _delegationClause;
+        private readonly AdtQueryBuilder _parent;
+        private SelectClause _clause;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SelectQuery"/> class.
```

Comment:
Don't use `<see />` to refer to the same class you are in. A self-reference isn't really helpful in Intellisense or web documentation.

You can just say "...of this class."

Reply:
We do this all over the place and it's the standard constructor doc comment. You can see it here too:
https://github.com/Azure/azure-sdk-for-net/blob/dac0f7e6d518b16e5bded4c947dae3053baada72/sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs#L42

Reply:
I don't disagree that it's not the most useful link though ...

Reply:
Then we should get rid of it in the ADT code too.

Think about this. You're on the web doc page for ADT object ctor, and it has a link to tell you more info about a type. You click that link and it takes you... right back to where you are.

That is confusing and weird, no?

Reply:
again, I agree
:D